### PR TITLE
[Docs] Fix math rendering in docs

### PR DESCRIPTION
**Pull request by hmellor**
I noticed that math rendering was broken.

I've updated it to use the plugin bundled with MkDocs Material as documented in https://squidfunk.github.io/mkdocs-material/reference/math/#mathjax.

You can verify that it's working by going to the Softmax section of the Paged Attention design doc.

### CODE REVIEW

**Review by DarkLight1337 — APPROVED** [review]
Thanks for fixing!
